.eslintrc.json

```diff
@@ -26,6 +26,9 @@
     "react/forbid-prop-types": 0,
     "import/prefer-default-export": 0,
     "class-methods-use-this": 0,
-    "no-plusplus": 0
+    "no-plusplus": 0,
+    "no-trailing-spaces": 0,
```

Comment:
i'd rather to keep this rule enabled (i actually find trailing spaces really annoying :-) ) , can u please delete this line?

Reply:
@jaanauati  Which rule?

Reply:
"no-trailing-spaces"

Reply:
@jaanauati I have enable it again!